packages/angular/golden/clr-angular.d.ts

```diff
@@ -1125,8 +1125,8 @@ export declare class ClrIfError extends AbstractIfState {
 }
 
 export declare class ClrIfExpanded implements OnInit, OnDestroy {
-    get expanded(): boolean;
-    set expanded(value: boolean);
+    get expanded(): boolean | string;
```

Comment:
The getter is not part of an `@Input` so do we need the `string` type?

Reply:
I don't understand the question?

Reply:
getter and setters must return and work with the same type so they are coupled to each other.

```ts
 get expanded(): boolean | string {
    return this._expanded;
  }

  @Input('clrIfExpanded')
  set expanded(value: boolean | string) {
    if (typeof value === 'boolean') {
      this.expand.expanded = value;
      this._expanded = value;
    }
  }
```

Even that the getter method is on top of the `Input` they are linked